app/src/components/Common/TextArea/TextArea.module.scss

```diff
@@ -16,14 +16,21 @@
   border-bottom: solid 1px;
   border-left: solid 1px;
   border-color: var(--disabled);
-  background-color: var(--backgroundColor);
   display: block;
   padding: 0px 10px 0 10px;
   @include regularText;
   resize: vertical;
   margin-top: 10px;
 }
 
+.darkBackground {
+  background-color: var(--backgroundColor);
+}
+
+.whiteTextArea {
+  color: black;
+}
+
```

Comment:
Ville gitt disse samme men "omvendt" navn.
`darkTextArea`
og 
`whiteTextArea` eller noe sånt. Som viser at de er på en måte er inverse av hverandre og den ene brukes dersom den andre ikke gjør det.

Reply:
Den nederste brukes for å gjøre teksten mørk når det er hvit bakgrunn på skjermen og textArea. Mener du at denne skal beholde navnet og den som heter dark background skal bytte til darkTextArea? Ble litt forvirret av "motsatt"😬